testapp/build.gradle

```diff
@@ -143,6 +143,7 @@ dependencies {
     } else {
         implementation project(':analytics')
     }
+    implementation project(':admob')
```

Comment:
Should we document how to use this latest admob in hostapp? 🤔

Reply:
@khairul-alam-licon  not yet.. after @corycaywood 's approval i will add it in userguide and changelog, thanks

Reply:
I don't agree @munir-rakuten, we should not merge this PR before it is documented / added in changelog. If you think this PR does not need documentation because it might not pass the peer validation, submit it as a draft PR ;)

Reply:
@Climbatize got your point 🙂, thanks

Reply:
Thanks for the addition @munir-rakuten, much better :)